identity-apps-core/apps/authentication-portal/src/main/resources/org/wso2/carbon/identity/application/authentication/endpoint/i18n/Resources.properties

```diff
@@ -148,7 +148,7 @@ information=Information
 user.details.submitted=User details successfully submitted
 close=Close
 other.login.options=Other login options
-sign.in.with=Sign In With
+sign.in.with=Sign in with
```

Comment:
Let's make this consistent with other places.

Reply:
Fixed with 8f324ad5fc35eed7633678a7feda80fea830554a